apps/librelingo_fakes/librelingo_fakes/fakes.py

```diff
@@ -192,6 +199,7 @@
     emptySkill,
     Skill(
         name="Mammals and birds",
+        filename="mammals_and_birds.yaml",
```

Comment:
should these file names not be relative to the course root, rather than the module folder?

Reply:
yes, though in the fakes it apparently does not matter. I'll update anyway

Reply:
yeah in the fakes it won't matter right now. I think it's better just for how it will appear in a text failure for example. If someone sees a test failure and they see the path does not look normal they might be distracted by that